pkg/apis/search/cache_types.go

```diff
@@ -1,7 +1,7 @@
 package search
 
 import (
-	corev1 "k8s.io/api/core/v1"
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	
	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
```
Group imports like this. <std> --> <third-party> --> <Karmada>